resources/user-agents/browsers/mobile-safari-uiwebview/mobile-safari-uiwebview.json

```diff
@@ -66,15 +73,30 @@
             "iPod": "iPod Touch"
```

Comment:
From what I recall, there were a lot of of mixed ones for this pattern. Let me look through the logs a bit and see if patterns can be reduced here at all.